sdk/program/src/vote/state/mod.rs

```diff
@@ -86,6 +90,18 @@ impl Lockout {
     pub fn is_locked_out_at_slot(&self, slot: Slot) -> bool {
         self.last_locked_out_slot() >= slot
     }
+
+    pub fn slot(&self) -> Slot {
+        self.slot
+    }
+
+    pub fn confirmation_count(&self) -> u32 {
+        self.confirmation_count
+    }
+
+    pub fn increase_confirmation_count(&mut self, by: u32) {
+        self.confirmation_count = self.confirmation_count.saturating_add(by)
```

Comment:
NOTE: switch to saturating from unchecked math here

Reply:
oh uhh... yeah looks like i missed a couple instances of same-module-pub.  will update. thanks!

Reply:
https://github.com/solana-labs/solana/pull/29753#discussion_r1080664805